ext/dom/tests/modern/html/serializer/HTMLDocument_serialize_text_03.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+DOM\HTMLDocument serialization escape text 03 - special tags in namespace should encode content
+--EXTENSIONS--
+dom
+--FILE--
+<?php
+
+$dom = DOM\HTMLDocument::createEmpty();
+$body = $dom->appendChild($dom->createElement("body"));
+foreach (["style", "script", "xmp", "iframe", "noembed", "noframes", "plaintext", "noscript"] as $tag) {
+    $tag = $body->appendChild($dom->createElementNS("some:ns", $tag));
+    $tag->textContent = "&\"<>\xc2\xa0 foobar";
+    $body->appendChild(new DOMText("\n"));
+}
+echo $dom->saveHTML();
+
+?>
+--EXPECT--
+<body><style>&amp;"&lt;&gt;&nbsp; foobar</style>
```

Comment:
Yes, the string escape algorithm it not in "attribute mode", so these must be proper `"` (otherwise this would also break `url("foo")` in CSS for `<style>` tags)